content/security/docs/data.md

```diff
@@ -1,7 +1,9 @@
 # Data encryption and secrets management
 
 ## Encryption at rest
-There are three different AWS-native storage options you can use with Kubernetes: [EBS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html), [EFS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEFS.html), and [FSx for Lustre](https://docs.aws.amazon.com/fsx/latest/LustreGuide/what-is.html).  All three offer encryption at rest using a service managed key or a customer master key (CMK). For EBS you can use the in-tree storage driver or the [EBS CSI driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver).  Both include parameters for encrypting volumes and supplying a CMK.  For EFS, you can use the [EFS CSI driver](https://github.com/kubernetes-sigs/aws-efs-csi-driver), however, unlike EBS, the EFS CSI driver does not support dynamic provisioning.  If you want to use EFS with EKS, you will need to provision and configure at-rest encryption for the file system prior to creating a PV. For further information about EFS file encryption, please refer to [Encrypting Data at Rest](https://docs.aws.amazon.com/efs/latest/ug/encryption-at-rest.html). Besides offering at-rest encryption, EFS and FSx for Lustre include an option for encrypting data in transit.  FSx for Luster does this by default.  For EFS, you can add transport encryption by adding the `tls` parameter to `mountOptions` in your PV as in this example: 
+There are three different AWS-native storage options you can use with Kubernetes: [EBS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html), [EFS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEFS.html), and [FSx for Lustre](https://docs.aws.amazon.com/fsx/latest/LustreGuide/what-is.html).  All three offer encryption at rest using a service managed key or a customer master key (CMK). For EBS you can use the in-tree storage driver or the [EBS CSI driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver).  Both include parameters for encrypting volumes and supplying a CMK.  For EFS, you can use the [EFS CSI driver](https://github.com/kubernetes-sigs/aws-efs-csi-driver). As of 23 March 2021, EFS CSI supports both [static and dynamic provisioning](https://aws.amazon.com/blogs/containers/introducing-efs-csi-dynamic-provisioning/)
```

Comment:
I see you removed the blurb on EFS's lack of support for dynamic provisioning. EFS can dynamically provision Access Points but it is still incapable of dynamically creating file systems. I think we need to be clear about that.

Reply:
Agree with @jicowan, this seems to be a slight misunderstanding. In the [AWS Blog post you added](https://aws.amazon.com/blogs/containers/introducing-efs-csi-dynamic-provisioning) they assume you have an existing EFS Filesystem provisioned already.

The dynamic provisioning feature creates EFS Access Points in the given filesystem when it detects a Persistent Volume Claim request. These get their own UID:GID and a dedicated directory in the Filesystem (with defaults it looks like `/dynamic_provisioning/pvc-12341231234...`). More details [in the official example](https://github.com/kubernetes-sigs/aws-efs-csi-driver/tree/master/examples/kubernetes/dynamic_provisioning).

I suggest keeping the original language but clarifying this a bit.

Reply:
@sebastianlzy can you revise the PR to make this clearer?